packages/design-system/src/components/Form/Primitives/_mixins.scss

```
@@ -82,9 +82,13 @@ $standardInputHeight: calc(#{tokens.$coral-sizing-m} - 0.2rem);
     box-shadow: 0 0 0 0.1rem tokens.$coral-color-accent-border;
   }
 
-  &.border_error {
+  &_borderError {
```

Feels like a BEM modifier, also I mostly saw hyphenated classes in Talend's projects 

```suggestion
  &--border-error {
```

---

yes, but CSS modules and BEM are not great friends. I want to keep autocompletion of classNames in the CSS. Using the `styles["style--name"]` syntax robs me of that.